nautobot/core/urls.py

```diff
@@ -16,6 +16,7 @@
     # Base views
     path("", HomeView.as_view(), name="home"),
     path("search/", SearchView.as_view(), name="search"),
+    path("modals/", ThemeView.as_view(), name="modals"),
```

Comment:
I don't quite understand where this is used - can you explain? Isn't the modal already present in the base view, so what's the case where we need to load it on the fly?

Reply:
This is a good catch.  I believe this was left over from a previous way I attempted to load the themes, but is no longer needed.  I'll remove it.